packages/starknet-snap/src/rpcs/switch-network.test.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  InvalidParamsError,
+  UserRejectedRequestError,
+} from '@metamask/snaps-sdk';
+import type { constants } from 'starknet';
+
+import { Config } from '../config';
+import { NetworkStateManager } from '../state/network-state-manager';
+import type { Network } from '../types/snapState';
+import {
+  STARKNET_SEPOLIA_TESTNET_NETWORK,
+  STARKNET_MAINNET_NETWORK,
+} from '../utils/constants';
+import { prepareConfirmDialog } from './__tests__/helper';
```

Comment:
```suggestion
import {
  UserRejectedOpError,
  InvalidNetworkError,
  InvalidRequestParamsError,
} from '../utils/exceptions';
import { prepareConfirmDialog } from './__tests__/helper';
```

Reply:
thank you, but it was expecting to change after prev PR merge, sorry i should put this PR into draft first